index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Alex Portfolio</title>
+</head>
+<body>
+    <!--Principal nav-->
+    <header>
+        <nav>
+          <a href="">Alexander</a>
+          <ul>
+            <li class="nav-list menu-collapsed" ><a href="">
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+            </a></li>
+          </ul>
+          
+          <ul class="navbar-items">
+            <li class="nav-list"><a href="">Portfolio</a></li>
+            <li class="nav-list"><a href="">About</a></li>
+            <li class="nav-list"><a href="">Contact</a></li>
+           
+          </ul>
+        </nav>
+    </header>
+
+        <!--main-->
+        <main id="main">
+            <!--home page-->
+            <section class="text-headline">
+              <h1 class="primery-text">Hey There.<br />I'm Alexander</h1>
+      
+              <div class="main-description">
+                <h2 class="supporting-text">I'm a Software Developer</h2>
+      
+                <p class="description">
+                  I can help you build a product, feature or website Look through some
+                  of my work and experiece! If you like what you see and have a
+                  project you need coded, don't hesitate to contact me.
+                </p>
+              </div>
```

Comment:
- [ ] As the previous reviewer requested, please try to adjust your background image to conform to the figma design 🙏 
### Your design
![image](https://user-images.githubusercontent.com/46839548/164548121-448822c3-f8f9-4cbc-b311-fd606e6a147c.png)
### Figma Design
![image](https://user-images.githubusercontent.com/85095702/164451271-ba1c6353-546f-4a05-a425-ac26bb9edb64.PNG)

Reply:
### @codepantha 
hello [screen shoot](https://drive.google.com/file/d/1R1b3x3CzzRYjBuNd8KTpHPRhdjAOA6wr/view?usp=sharing).
Due to the screen size the position can change. you can see here de result that i have from my computer.
waitting for your feed back

### @phelian23 
Please can you take a look

Reply:
Alright @phelian23. I still think the placement is not the best in smaller devices like `320px` and `375px` but we can work with this. Please go ahead and request for another review.

Reply:
i have made my last review request